wrims-comparison-test/src/test/java/wrims/compare/stepdefinitions/AzureStepDefinitions.java

```diff
@@ -0,0 +1,81 @@
+package wrims.compare.stepdefinitions;
```

Comment:
We are in the midst of a rename of packages to capture the TLD package name in:
https://github.com/CentralValleyModeling/wrims-engine/discussions/154
https://github.com/CentralValleyModeling/wrims-engine/pull/156

These package names do not follow this new naming. @zack-rma  @jmdegeorge How will this be resolved?